comma/individual.py

```diff
@@ -113,20 +186,26 @@ def sampling_from_ipf(size: int, dir_params: str) -> pd.DataFrame:
         df_weights = pd.read_csv(fpath_weights, sep=",", index_col=0)
         weights = df_weights["weight"] / df_weights["weight"].sum()
         indices = df_weights.index
-        sample_indices = np.random.choice(indices, size, p=weights)
+        # use the new random method of numpy
+        if rng is None:
+            rng = np.random.default_rng(12345)
```

Comment:
As above, I would parametrize the random seed to make it easier to change in all places.
```suggestion
            rng = np.random.default_rng(RANDOM_SEED)
```